app/scripts/modules/ecs/src/serverGroup/configure/wizard/capacityProvider/CapacityProvider.tsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import { module } from 'angular';
+import { react2angular } from 'react2angular';
+import {IEcsCapacityProviderStrategy, IEcsServerGroupCommand} from '../../serverGroupConfiguration.service';
+import { HelpField, withErrorBoundary } from  '@spinnaker/core';
+
+export interface ICapacityProviderProps {
+  command: IEcsServerGroupCommand;
+  notifyAngular: (key: string, value: any) => void;
+  configureCommand: (query: string) => PromiseLike<void>;
+}
+
+interface ICapacityProviderState {
+  capacityProviderStrategy: IEcsCapacityProviderStrategy[],
+}
+
+class CapacityProvider extends React.Component<ICapacityProviderProps, ICapacityProviderState>{
```

Comment:
can we preface this component name with "Ecs", since we're exporting it into the context of the larger app? (see [this convo](https://github.com/spinnaker/deck/pull/8699#pullrequestreview-521950184))

Reply:
Understood the context from the conversation, thanks for the convo link. I will change it to preface with "Ecs".

Reply:
@paragbhingre I saw that this change was made earlier, but it seems to have been reverted. Can you add that back and sync from /master ahead of a final review?